architecture/core_concepts/routes.adoc

```diff
@@ -12,41 +12,86 @@ toc::[]
 
 == Overview
 
-An OpenShift route is a way to expose a
-link:pods_and_services.html#services[service] by giving it an
-externally-reachable hostname like `www.example.com`.
-
-A defined route and the endpoints identified by its service can be consumed by a
-router to provide named connectivity that allows external clients to reach your
-applications. Each route consists of a route name (limited to 63 characters),
-service selector, and (optionally) security configuration.
+An {product-title} xref:../../architecture/core_concepts/routes.adoc#[route] exposes a
+xref:../../architecture/core_concepts/pods_and_services.adoc#services[service] at a
+host name, like _www.example.com_, so that external clients can reach it by
+name.
+
+DNS resolution for a host name is handled separately from routing.
+Your administrator may have configured a
+xref:../../install_config/install/prerequisites.adoc#prereq-dns[DNS wildcard entry]
+that will resolve to the {product-title} node that is running the
+{product-title} router. If you are using a different host name you may
+need to modify its DNS records independently to resolve to the node that
+is running the router.
+
+Each route consists of a route name (limited to 63 characters), service selector,   
+and (optionally) security configuration.
 
 == Routers
 
-An OpenShift administrator can deploy _routers_ in an OpenShift cluster, which
-enable routes link:../../dev_guide/routes.html#creating-routes[created by
-developers] to be used by external clients. The routing layer in OpenShift is
-pluggable, and two available link:#available-router-plug-ins[router plug-ins]
-are provided and supported by default.
+An {product-title} administrator can deploy _routers_ to nodes in an
+{product-title} cluster, which enable routes
+xref:../../dev_guide/routes.adoc#creating-routes[created by developers]
+to be used by external clients.
+
+
+The routing layer in {product-title} is
+pluggable, and two available xref:#available-router-plug-ins[router plug-ins]
+are provided and supported.
 
 ifdef::openshift-enterprise,openshift-origin[]
 [NOTE]
 ====
-See the link:../../install_config/install/deploy_router.html[Installation and
+See the xref:../../install_config/install/deploy_router.adoc#[Installation and
 Configuration] guide for information on deploying a router.
 ====
 endif::[]
 
-OpenShift routers provide external host name mapping and load balancing
-to link:pods_and_services.html#services[services] over protocols that
+A router uses the service selector to find the service and the endpoints backing
+the service. Service-provided load balancing is bypassed and replaced with the
+router's own load balancing. Routers watch the cluster API and automatically
+update their own configuration according to any relevant changes to the IP
+addresses of pods that provide the service.  Routers may be containerized.
+Custom routers can be deployed to communicate modifications of API objects to
```

Comment:
Can we indicate somehow that there are other reasons you may want to customize a router?